pkg/core/database/heavy/transactions.go

```diff
@@ -249,6 +251,19 @@ func (t transaction) FetchOutputExists(destkey []byte) (bool, error) {
 	return exists, err
 }
 
+// FetchOutputUnlockHeight returns the unlockheight of an output
```

Comment:
In both dusk-blockchain (incl CLI help) and dusk-wallet, we use mostly `lockTime/lock`. Would it make sense to refactor this and use everywhere UnlockHeight to make it more readable, unless we are going to support both lock-by-height and lock-by-timestamp?

Reply:
Good point. I think, with Phoenix, we will only support lock-by-height, so it would probably make sense to refactor.

Reply:
Your question actually got me to realize a terrible error I made: LockTime is not the same as UnlockHeight. I've adjusted the naming of these functions/fields and how they are writting into their databases for both `dusk-blockchain` and `dusk-wallet`. Once dusk-blockchain/dusk-wallet#19 is merged, I can finalize this fix. Hopefully, the naming should then no longer be confusing.

For clarity:
- `LockTime` is the amount of blocks to lock an output up for
- `UnlockHeight` is the absolute blockchain height at which the output becomes unlocked